src/pin-field/PinField.stories.ts

```diff
@@ -31,6 +32,7 @@ export const Interactive: ComponentStoryFormat<Args> = {
       data-testid="test-pin-field"
       label="${ifNotEmpty(args.label)}"
       value="${args.value}"
+      max-length=${args.maxLength}
```

Comment:
Use `ifNotEmpty` or `ifDefined` on the Interactive story to allow for removing max length in Live Property Editor
```suggestion
      max-length=${ifNotEmpty(args.maxLength)}
```

Reply:
Used ifDefined here since this.maxLength is of type number thus we can't use ifNotEmpty